vermouth/gmx/topology.py

```diff
@@ -127,13 +136,18 @@ def write_gmx_topology(system, top_path, itp_paths=[], C6C12=False, defines=(),
     if not system.molecules:
         raise ValueError("No molecule in the system. Nothing to write.")
 
+    itp_paths = itp_paths[::-1]
     include_string = ""
-    # First we write the atomtypes and nonbondparams directive
-    if itp_paths:
-        write_atomtypes(system, itp_paths[0], C6C12)
-        write_nonbond_params(system, itp_paths[1], C6C12)
-        include_string = "\n".join('#include "{}"'.format(path) for path in itp_paths[:2])
-        include_string += "\n"
+    # First we write the atomtypes directive
+    if "atomtypes" in system.gmx_topology_params:
+        _path = itp_paths.pop()
+        write_atomtypes(system, _path, C6C12)
+        include_string += f'\n #include "{_path}"'
+    # Next we write the nonbond_params directive
+    if "nonbond_params" in system.gmx_topology_params:
+        _path = itp_paths.pop()
+        write_nonbond_params(system, _path, C6C12)
+        include_string += f'\n #include "{_path}"\n'
```

Comment:
I guess we need the itp_paths because of the multichain Go thing? Or could we generate sensible names by doing `_path = f"{system.molecules[0].meta['moltype']}_atomtypes.itp"` or something along those lines?

Reply:
shall we make it a defaultdict instead? That is each nonbonded section gets a default file name assigned as dict? Perhaps more clear than a list

Reply:
I think that would  be a great improvement